### DISCUSSION TITLE
Mutation not working

### DISCUSSION THREAD

**Discussion by arkodeepg** [EvgeniyMukhamedjanov/liquid-ajax-cart#88] [Q&A] [answered]
Hi Guys, I'm not able to configure mutation.
I have tried to configure it but defining it outside the conf as
`window.liquidAjaxCart.conf('mutations', [
  myMutation
]);`
but its still not working and giving error. Any tips on how to resolve this? 

![image](https://github.com/EvgeniyMukhamedjanov/liquid-ajax-cart/assets/43224169/657da096-15e7-4294-926e-ee69b860bef2)
![image](https://github.com/EvgeniyMukhamedjanov/liquid-ajax-cart/assets/43224169/309f0ff9-5a97-4d10-b8b6-625d88b96bb8)

**Comment by EvgeniyMukhamedjanov** [comment, marked as answer]
Hi @arkodeepg 
Could you make sure you use the latest version of Liquid Ajax Cart, as the mutation feature appeared only in the latest update v2.1.0.
If you use the latest version, pls share a preview link of your store, I'll check what might go wrong :-)

**Comment by arkodeepg** [comment]
Thank you @EvgeniyMukhamedjanov I was on an older version (2.0.1) so it was causing issue, once I updated it seem to be working, I'm still checking but thank you again.

**Reply by EvgeniyMukhamedjanov** [reply]
Happy to help!
I noticed, you set conf parameters from the runWhenInit function which is, I assume, is executed on the liquid-ajax-cart:init event.
The best practice is to set up conf parameter rather before init -- right after liquid ajax cart library is included to the page, so that when liquid ajax cart is initialized, it would already have info about all the mutations that should be applied.

**Comment by arkodeepg** [comment]
Got it thanks

**Reply by arkodeepg** [reply]
answered via email didnt't know it would send the whole thread 😅. Anyways again thank you this was where I was stuck.

**Reply by EvgeniyMukhamedjanov** [reply]
No worries :-)
